Issues deploying using wrangler2

There are a couple of warnings and an error that shows up when using the new wrangler CLI.

```
wrangler publish

 ⛅️ wrangler 2.0.28
--------------------
▲ [WARNING] Processing wrangler.toml configuration:

    - 😶 Ignored: "type":
      Most common features now work out of the box with wrangler, including modules, jsx,
  typescript, etc. If you need anything more, use a custom build.
    - Deprecation: "zone_id":
      This is unnecessary since we can deduce this from routes directly.
    - The "route" field in your configuration is an empty string and will be ignored.
      Please remove the "route" field from your configuration.



✘ [ERROR] Missing entry-point: The entry-point should be specified via the command line (e.g. `wrangler publish path/to/script`) or the `main` config field.
```

--- a/README.md
+++ b/README.md
@@ -14,10 +14,9 @@ An example of stateless Discord Bot using Slash Commands feature and Cloudflare 
 1. Clone this repository.
 2. Copy `wrangler.example.toml`, then rename it to `wranger.toml` .
 3. Edit the file, filling `account_id` and `vars.PUBLIC_KEY` .
-4. Build the current package and all its dependencies using `cargo b -r` (Release version)
-5. Deploy using `wrangler publish` .
-6. Register your endpoint at Discord Developer Portal.
-7. Create an application command. For example:
+4. Deploy using `wrangler publish` .
+5. Register your endpoint at Discord Developer Portal.
+6. Create an application command. For example:
    ```console
    $ curl \
      -H "Authorization: Bot ${DISCORD_BOT_TOKEN}" \
@@ -25,4 +24,4 @@ An example of stateless Discord Bot using Slash Commands feature and Cloudflare 
      -d '{"name":"hello","description":"The bot will say \"Hello, world!\"."}' \
      "https://discord.com/api/v8/applications/${APPLICATION_ID}/commands"
    ```
-1. Done!
+8. Done!

--- a/README.md
+++ b/README.md
@@ -12,11 +12,12 @@ An example of stateless Discord Bot using Slash Commands feature and Cloudflare 
 
 ## 📦 Installation
 1. Clone this repository.
-1. Copy `wrangler.example.toml`, then rename it to `wranger.toml` .
-1. Edit the file, filling `account_id` and `vars.PUBLIC_KEY` .
-1. Deploy using `wrangler publish` .
-1. Register your endpoint at Discord Developer Portal.
-1. Create an application command. For example:
+2. Copy `wrangler.example.toml`, then rename it to `wranger.toml` .
+3. Edit the file, filling `account_id` and `vars.PUBLIC_KEY` .
+4. Build the current package and all its dependencies using `cargo b -r` (Release version)
+5. Deploy using `wrangler publish` .
+6. Register your endpoint at Discord Developer Portal.
+7. Create an application command. For example:
    ```console
    $ curl \
      -H "Authorization: Bot ${DISCORD_BOT_TOKEN}" \

--- a/wrangler.example.toml
+++ b/wrangler.example.toml
@@ -1,9 +1,8 @@
 name = "stateless-discord-bot"
-type = "rust"
 account_id = "{{ACCOUNT_ID}}"
 workers_dev = true
-route = ""
-zone_id = ""
+compatibility_date = "2022-08-04"
+main="./worker/worker.js"
 
 [vars]
 PUBLIC_KEY = "{{PUBLIC_KEY}}"

